sqlserver/datadog_checks/sqlserver/config_models/instance.py

```diff
@@ -152,7 +152,6 @@ class InstanceConfig(BaseModel):
     host: str
     ignore_missing_database: Optional[bool] = None
     include_ao_metrics: Optional[bool] = None
-    include_db_file_space_usage_metrics: Optional[bool] = None
```

Comment:
I recommend keeping the configuration option, but defaulting it to True, so this allows users to turn the feature off if need be.